src/bluecadet/core/SettingsManager.cpp

```diff
@@ -51,6 +51,7 @@ void SettingsManager::setup(ci::app::App::Settings * appSettings, ci::fs::path j
 	addCommandLineParser("mouse", [&](const string &value) { mMouseEnabled = value == "true"; });
 	addCommandLineParser("tuio", [&](const string &value) { mTuioTouchEnabled = value == "true"; });
 	addCommandLineParser("native", [&](const string &value) { mNativeTouchEnabled = value == "true"; });
+	addCommandLineParser("multiNativeTouchScreen", [&](const string &value) { mMultiNativeTouchScreenEnabled = value == "true"; });
```

Comment:
Hm I feel like this should default to true if this value doesn't exist; Or we should flip the naming to something like `mForceCinderTouch`. What do you think @shi-weili ? My main thinking is that your driver seems more helpful, and the Cinder driver should just be a fallback.

Reply:
Discussed renaming to `supportMultipleNativeTouchScreens`. Verbose but more clear.